docs/source/ja/_toctree.yml

```diff
@@ -134,3 +134,24 @@
   - local: model_memory_anatomy
     title: モデルトレーニングの解剖学
   title: コンセプチュアルガイド
+  - sections:
+    - local: internal/modeling_utils
+      title: カスタムレイヤーとユーティリティ
+    - local: internal/pipelines_utils
```

Comment:
Your local filename is `pipeline_utils`, but it should be renamed to match whats in the `toctree` (`pipelines_utils`).